test/e2e-tests.sh

```diff
@@ -31,7 +31,7 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 # Build kn before integration tests, so we fail fast in case of error.
 function cluster_setup() {
   header "Building client"
-  go build -v -mod=vendor ./cmd/... || return 1
+  $(dirname $0)/../hack/build.sh -u || return 1
```

Comment:
I have got build github.com/knative/client/cmd/kn: cannot load k8s.io/api/admissionregistration/v1alpha1 locally with change. Do you know how to fix it? @rhuss

Reply:
OK. I have resolved it. However, do you know it would be more straightforward to add
export GO111MODULE=on above Line 34, since $(dirname $0)/../hack/build.sh -u would go through 🕸️  Update 🚧 Compile 🧪  Test 🧹  Format and 📖 Docs. What we only need here is the compile.